src/main/java/org/folio/util/ResponseUtils.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.util;
+
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import java.net.URI;
+
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.persist.PgExceptionUtil;
+
+import io.vertx.core.Future;
+import io.vertx.ext.web.handler.HttpException;
+
+public class ResponseUtils {
```

Comment:
**Info**
`ResponseUtil` class works on `Promises` and is more complex. We are using `ResponseUtils` in other modules